### ISSUE TITLE
HTML Table support

### ISSUE THREAD

**Issue by jasonfoy99** [jpuri/react-draft-wysiwyg#323]
This is a really fantastic project. The only thing missing here is table support. I know the author is busy working on something else before she has chance of digging in. I am really new to react and draft. I am just wondering if there is any workaround to achieve table support, such as including a plugin like draft-js-table-plugin to work with react-draft-wysiwyg. Has anyone get HTML table working with react-draft-wysiwyg?

**Comment by jpuri** [comment]
Hi @jasonfoy99 , thanks you liked editor project.

Its hard to use draft-js-table-plugin with react-draft-wysiwyg.

A way to achieve it would be create custom block renderer and custom toolbar button.

There are quite a few people who need it and it very useful feature. I am thinking of developing it - but it will be comparatively bigger task and since I am also busier these day - I can not promise how soon it will be ready.

**Comment by jasonfoy99** [comment]
Understood. Thank you for your quick reply.

**Comment by crazywako** [comment]
Any progress on this?

**Comment by bevinhex** [comment]
Any progress?

**Comment by Shadowman4205** [comment]
Is there any progress?

**Comment by tomkiss** [comment]
Just gonna add my name to the list 👍

**Comment by tanmynguyen** [comment]
Yep! That bigger feature. I need it :(. I hope @jpuri will resolve it .

**Comment by couturecraigj** [comment]
@jpuri do you want me to build a PR for this because I would like to see this as well.

**Comment by jpuri** [comment]
Sure, I will be very happy for a PR of this feature. Plz let me know if I can help you anywhere in doing that.

**Comment by couturecraigj** [comment]
So,

I am not sure about how you want the table to look for the SVG icon.  Maybe I can leave that to you.  I will try to get the normal functionality working and we can go from there.

**Comment by jpuri** [comment]
That wounds good 👍

**Comment by couturecraigj** [comment]
I have been trying to figure this out... Everything seems to be in
different places.  I do not know how I would go about doing this.  I think
I am giving up.  I want to help but, I think I might just try to build my
own with a basic draft-js then find out how to integrate it.
Unfortunately, that will take some time.

What I see as important to get this up and running is the following:
- Add table button
- Choose the size of the table (height x width)
- While in a table add column
- While in a table add row
- While in a table remove column
- While in a table remove row

I will see if I can revisit this after I implement it individually.

**Comment by wliadmin** [comment]
Hi,

Any update on this? I really need this plugin or way to use table in the editor.

**Comment by vanhuy12toan2** [comment]
1. asdfsad
sdfsadf

**Comment by BjoernRave** [comment]
Anyone has a suggestion what is the best way to achieve this right now?

**Comment by yomi05** [comment]
Is there any update on this... really need the feature

**Comment by milkysingh** [comment]
@jpuri ANY UPDATES?

**Comment by DamilolaStem** [comment]
Any update on how to implement table in react draft ?

**Comment by mauriciomutte** [comment]
Is there any progress?

**Comment by tinhlv** [comment]
I am also looking forward to it.
I am using the components available in plugin 
draft-js-table-plugin to include them in my project.
![Screen Shot 2021-01-14 at 14 40 58](https://user-images.githubusercontent.com/16912951/104559437-f6b32f80-5676-11eb-8e8a-38bd66045b66.png)

**Comment by zhuhongyin** [comment]
> draft-js-table-plugin

@tinhlv looks good, do you have a guide for how to use draft-js-table-plugin

**Comment by SangeetAgarwal** [comment]
@tinhlv same question as @zhuhongyin  any guidance on how you got this to work as you showed in the image? Thanks for the help.

**Comment by zackiieonebbad** [comment]
No

**Comment by anandgautamvns** [comment]
How can i use this

**Comment by mmpatel78** [comment]
Hi... I want to use table in the rich text editor.. is there anyway to use table.. please let me know

**Comment by sandstone991** [comment]
Any updates on this?

**Comment by razlani** [comment]
I daresay we aren't seeing any update to this repo any time soon.. table-seekers seek elsewhere.

**Comment by majadul1989** [comment]
> I am also looking forward to it. I am using the components available in plugin draft-js-table-plugin to include them in my project. ![Screen Shot 2021-01-14 at 14 40 58](https://user-images.githubusercontent.com/16912951/104559437-f6b32f80-5676-11eb-8e8a-38bd66045b66.png)

Its just on Image for this screenshot I think

**Comment by caoss** [comment]
Any updates on this?

**Comment by qzw881130** [comment]
any updates ?

**Comment by QiaoLi1996** [comment]
Is there any progress?

**Comment by hamidrezaramzani** [comment]
any update?

**Comment by vishal-infiniqe** [comment]
any update @jpuri ?
